File: app/src/main/java/com/example/nickolas/vk/models/enteties/Message.java
package com.example.nickolas.vk.models.enteties;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;


public class Message {

    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("date")
    @Expose
    private Integer date;
    @SerializedName("out")
    @Expose
    private Integer out;
    @SerializedName("user_id")
    @Expose
    private Integer userId;
    @SerializedName("read_state")
    @Expose
    private Integer readState;
    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("body")
    @Expose
    private String body;

    @SerializedName("emoji")
    @Expose
    private Integer emoji;
    @SerializedName("chat_id")
    @Expose
    private Integer chatId;
    @SerializedName("chat_active")
    @Expose
    private List<Integer> chatActive = null;
    @SerializedName("users_count")
    @Expose
    private Integer usersCount;
    @SerializedName("admin_id")
    @Expose
    private Integer adminId;
    @SerializedName("random_id")
    @Expose
    private Integer randomId;


    @SerializedName("photo_50")
    @Expose
    private String photo50;
    @SerializedName("photo_100")
    @Expose
    private String photo100;
    @SerializedName("photo_200")
    @Expose
    private String photo200;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getDate() {
        return date;
    }

    public void setDate(Integer date) {
        this.date = date;
    }

    public Integer getOut() {
        return out;
    }

    public void setOut(Integer out) {
        this.out = out;
    }

    public Integer getUserId() {
        return userId;
    }

    public void setUserId(Integer userId) {
        this.userId = userId;
    }

    public Integer getReadState() {
        return readState;
    }

    public void setReadState(Integer readState) {
        this.readState = readState;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }

    public Integer getEmoji() {
        return emoji;
    }

    public void setEmoji(Integer emoji) {
        this.emoji = emoji;
    }

    public Integer getChatId() {
        return chatId;
    }

    public void setChatId(Integer chatId) {
        this.chatId = chatId;
    }

    public List<Integer> getChatActive() {
        return chatActive;
    }

    public void setChatActive(List<Integer> chatActive) {
        this.chatActive = chatActive;
    }

    public Integer getUsersCount() {
        return usersCount;
    }

    public void setUsersCount(Integer usersCount) {
        this.usersCount = usersCount;
    }

    public Integer getAdminId() {
        return adminId;
    }

    public void setAdminId(Integer adminId) {
        this.adminId = adminId;
    }

    public Integer getRandomId() {
        return randomId;
    }

    public void setRandomId(Integer randomId) {
        this.randomId = randomId;
    }


    public String getPhoto50() {
        return photo50;
    }

    public void setPhoto50(String photo50) {
        this.photo50 = photo50;
    }

    public String getPhoto100() {
        return photo100;
    }

    public void setPhoto100(String photo100) {
        this.photo100 = photo100;
    }

    public String getPhoto200() {
        return photo200;
    }

    public void setPhoto200(String photo200) {
        this.photo200 = photo200;
    }
}

File: app/src/main/java/com/example/nickolas/vk/models/enteties/Message.kt
package com.example.nickolas.vk.models.enteties

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class Message {

    @SerializedName("id")
    @Expose
    var id: Int? = null
    @SerializedName("date")
    @Expose
    var date: Long? = null
    @SerializedName("out")
    @Expose
    var out: Int? = null
    @SerializedName("user_id")
    @Expose
    var userId: Int? = null
    @SerializedName("read_state")
    @Expose
    var readState: Int? = null
    @SerializedName("title")
    @Expose
    var title: String? = null
    @SerializedName("body")
    @Expose
    var body: String? = null
    @SerializedName("emoji")
    @Expose
    var emoji: Int? = null
    @SerializedName("chat_id")
    @Expose
    var chatId: Int? = null
    @SerializedName("chat_active")
    @Expose
    var chatActive: List<Int>? = null
    @SerializedName("users_count")
    @Expose
    var usersCount: Int? = null
    @SerializedName("admin_id")
    @Expose
    var adminId: Int? = null
    @SerializedName("random_id")
    @Expose
    var randomId: Int? = null
    @SerializedName("photo_50")
    @Expose
    var photo50: String? = null
    @SerializedName("photo_100")
    @Expose
    var photo100: String? = null
    @SerializedName("photo_200")
    @Expose
    var photo200: String? = null
    var user: User? = null
}
